ChangeLog.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to
 
 ### Removed
 * The `global_search` flag in `freud.environment.EnvironmentCluster`.
+* Zero padding from properties of `freud.environment.EnvironmentCluster` and `freud.environment.EnvironmentMotifMatch`.
```

Comment:
This should be a “Changed” message. It’s not an API removal, it is a change in the API return convention and type.